database/src/backend/heed/database.rs

```diff
@@ -87,12 +150,54 @@ impl<T: Table> DatabaseRo<T> for HeedTableRo<'_, T> {
     fn get_range<'a, Range>(
         &'a self,
         range: Range,
-    ) -> Result<impl Iterator<Item = Result<T::Value, RuntimeError>> + 'a, RuntimeError>
+    ) -> Result<impl Iterator<Item = Result<(T::Key, T::Value), RuntimeError>> + 'a, RuntimeError>
```

Comment:
Something to note about including the `T::Key` by default is that `redb::AccessValue::access()` on the key will add just the slight bit more overhead.

Could have a `get_range_keys()` and `get_range_values()` I guess.

Reply:
Do we need to include the key?

Reply:
Not necessarily but most `get_range()` APIs return `(key, value)` (`heed`, `redb`, `BTreeMap`, etc).

I'm certain it would have use cases, if the overhead is a problem I don't think adding `get_range_{keys,values}()` would be that bad.

Reply:
I'm unsure, the only tables `get_range` should really be called are tables which have indexes for keys i.e. integers with no gaps.

It doesn't really make sense to use this on a table which has hashes for keys.

So the keys would already be known by the caller

Reply:
Reverted back to only values being returned.